__tests__/seed.test.js

```diff
@@ -1,64 +1,72 @@
 const data = require("../db/data/test-data");
-const seed = require("../db/seeds/seed")
-const db = require("../db/connection.js")
-const request = require("supertest")
-const app = require("../app.js")
-const endPoints = require('../endpoints.json')
+const seed = require("../db/seeds/seed");
```

Comment:
as we arent testing the seed function in this file, we can rename this to something like app.test.js